### ISSUE TITLE
[bug]Misplaced format If there are UTF-8 characters in the data

### ISSUE THREAD

**Issue by haptear** [minhhungit/ConsoleTableExt#30]
If there are UTF-8 characters in the data, it will cause the output to be misformatted

![error](https://user-images.githubusercontent.com/16282949/127624926-3bc00bdd-e6a4-4948-ab9b-7ab95338c68a.png)

**Comment by minhhungit** [comment]
@haptear can you give me your test code please

**Comment by haptear** [comment]
@minhhungit  the test code

using ConsoleTableExt;
using System;
using System.Collections.Generic;

namespace ConsoleApp6
{
    class Program
    {
        static void Main(string[] args)
        {
            var tableData =new List<object>() { new { Id="xxx", Name="tab其它语言test", Host="127.0.0.1", Port=80, status ="success" } };

            ConsoleTableExt.ConsoleTableBuilder
                .From<dynamic>(tableData)
                .WithTitle("ALL CENTERS", ConsoleColor.Yellow, ConsoleColor.DarkGray)
                .WithFormat(ConsoleTableBuilderFormat.Alternative)
                .ExportAndWriteLine(TableAligntment.Left);

            Console.ReadLine();
        }
    }
}

**Comment by minhhungit** [comment]
![image](https://user-images.githubusercontent.com/2279508/127632170-592cf8e5-388e-4bc0-ad4b-5e878a69b4fc.png)
this is first look, still checking more

**Comment by minhhungit** [comment]
After adding `Console.OutputEncoding = Encoding.Unicode;` I see the problem

**Comment by minhhungit** [comment]
![image](https://user-images.githubusercontent.com/2279508/127632606-616e404f-788d-45e6-9471-4f2100643251.png)

it looks it has extra space

**Comment by haptear** [comment]
Thank you for your quick reply

**Comment by minhhungit** [comment]
I have no idea, my lack of knowledge about this case

It counts length of text correctly, the text "tab其它语言test" has length 11, but after console print out, console need more space

**Comment by haptear** [comment]
The actual length of the string "tab其它语言test" is 15,UTF-8 characters should be *2

**Comment by haptear** [comment]
@minhhungit  You can get the actual length with "GetStringLen"
GetStringLen("tab其它语言test")==15

```
public static int GetStringLen(string str)
        {
            int i = 0;//count
            foreach (char newChar in str)
            {
                if ((int)newChar > 127)
                {
                    //utf-8 characters
                    i += 2;
                }
                else
                {
                    i++;
                }
            }
            return i;
        }
```

**Comment by minhhungit** [comment]
@haptear thank you about the suggestion but that is not only length issue but also windows console problem

Problem here for a text for example "tab其它语言test", we can get the length is 15, but windows app don't print the length exactly 

I tried but no lucky, I am quite sure this is problem of windows console

**Comment by minhhungit** [comment]
I found this post

https://stackoverflow.com/questions/26975736/why-is-the-length-of-this-string-longer-than-the-number-of-characters-in-it

**Comment by haptear** [comment]
@minhhungit  
hi can you merge request?

**Comment by minhhungit** [comment]
@haptear thanks, I will review the PR soon

**Comment by minhhungit** [comment]
released in 3.1.8
